src/state/routing/types.ts

```diff
@@ -87,6 +87,7 @@ export interface QuoteResult {
 export const INITIALIZED = 'Initialized'
 export const NO_ROUTE = 'No Route'
 
-export type GetQuoteResult = QuoteResult | typeof INITIALIZED | typeof NO_ROUTE
+export type GetQuoteError = typeof INITIALIZED | typeof NO_ROUTE
```

Comment:
so NO_ROUTE is technically an error i think? it gives us a 500 iirc. if we use rtk-query `query` instead of `queryFn` it will give that result in the `error` field

not sure about initialized - i feel like we could do something different there instead of returning a str, but that could be a discussion later

they're not valid quote results with routes, so they feel like errors in my mind 🤔